debian/changelog

```diff
@@ -1,5 +1,9 @@
 python3-oq-engine (3.23.0-1~xenial01) xenial; urgency=low
 
```

Comment:
You must move your 3 lines in the changelog up, i.e. above the line `python3-oq-engine (3.23.0-1~xenial01) xenial; urgency=low` since they will be part of engine 3.24